common/flatpak-installed-ref.c

```diff
@@ -423,6 +517,60 @@ flatpak_installed_ref_get_eol_rebase (FlatpakInstalledRef *self)
   return priv->eol_rebase;
 }
 
+/**
+ * flatpak_installed_ref_get_appdata_name:
+ * @self: a #FlatpakInstalledRef
+ *
+ * Returns the name field from the appdata.
+ *
```

Comment:
Should say that this string is localized and this returns the value for the current locale.